addons/necropost-highlighter/addon.json

```diff
@@ -0,0 +1,174 @@
+{
+  "name": "Necropost highlighter",
+  "description": "Spot old forum topics brought back to life, before opening the message. Topics started much earlier than others listed on the same page are highlighted (in selected forums). If you've wasted time reading zombie messages or a help/question/request that was actually asked years ago, this is for you!",
+  "info": [
+    {
+      "type": "notice",
+      "id": "whatGetsHighlighted",
+      "text": "Sometimes an older topic is kept alive for good reasons, like an ongoing conversation, or the original poster following up their own topic months later. All older topics in selected forums are highlighted, including occasional lingering topics that are not necroposts. (That's why they get highlighted, but not removed.)"
+    }
+  ],
+  "credits": [
+    {
+      "name": "RT_Borg",
+      "link": "https://github.com/RT-Borg"
+    }
+  ],
+  "dynamicDisable": true,
+  "dynamicEnable": true,
+  "enabledByDefault": false,
+  "tags": ["community", "forums"],
+  "versionAdded": "1.28.0",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["^https://scratch.mit.edu/discuss/(?:m/|)(?:search|1|3|4|6|7|9|11|49)/"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Highlight color",
+      "id": "highlightColor",
+      "type": "color",
+      "allowTransparency": true,
+      "default": "#D1E9F2"
+    },
+    {
+      "name": "How many topics is 'old'? (7,500 is about a month)",
```

Comment:
Maybe ask "How many months is 'old'?" and multiply that by 7500 in the code.

Reply:
I think that should look good, if I say "Roughly how many months is 'old'?"

"Roughly" should avoid users thinking it will be precisely a month (it could be more like 27 or 33 days, depending on recent forum traffic).

Putting the 7500 topic interval back in the code also gives us more flexibility to update the per-month interval if eg Scratch forums get significantly more traffic next year.

Thanks @RedGuy12. Commit incoming.